### PR TITLE
Remove sommelier dependency for xfce4-terminal

### PR DESCRIPTION
**Pull request by supechicken**
Part of #4771

Tested on `x86_64`

### CODE REVIEW

File: packages/xfce4_terminal.rb [hunk 1]
```diff
@@ -36,5 +37,13 @@ def self.build
...
   def self.install
     system "make install DESTDIR=#{CREW_DEST_DIR}"
+    system "cat <<'EOF'> xfce4-terminal
+WAYLAND_DISPLAY=wayland-0
+GDK_BACKEND=wayland
+DISPLAY=
+#{CREW_PREFIX}/bin/xfce4-terminal $@
+EOF"
+    FileUtils.mv "#{CREW_DEST_PREFIX}/bin/xfce4-terminal", "#{CREW_DEST_PREFIX}/bin/xfce4_terminal"
+    system "install -Dm755 xfce4-terminal #{CREW_DEST_PREFIX}/bin/xfce4-terminal"
```
Comment (uberhacker) [inline packages/xfce4_terminal.rb:47]: So we end up with `xfce4_terminal` and `xfce4-terminal`?  What is the difference?

File: packages/xfce4_terminal.rb [hunk 2]
```diff
@@ -36,5 +37,13 @@ def self.build
 
   def self.install
     system "make install DESTDIR=#{CREW_DEST_DIR}"
+    system "cat <<'EOF'> xfce4-terminal
+WAYLAND_DISPLAY=wayland-0
+GDK_BACKEND=wayland
+DISPLAY=
+#{CREW_PREFIX}/bin/xfce4-terminal $@
```
Comment (uberhacker) [inline packages/xfce4_terminal.rb:44]: `#{CREW_PREFIX}/bin/xfce4_terminal`?  (Replace `-` with `_`)

File: packages/xfce4_terminal.rb (hunk 1 above)
Comment (supechicken) [reply packages/xfce4_terminal.rb:47]: The new `xfce4-terminal` script will set the Wayland display variable at start

File: packages/xfce4_terminal.rb (hunk 2 above)
Comment (supechicken) [reply packages/xfce4_terminal.rb]: Done

File: packages/xfce4_terminal.rb [hunk 3]
```diff
@@ -36,5 +37,13 @@ def self.build
 
   def self.install
     system "make install DESTDIR=#{CREW_DEST_DIR}"
+    system "cat <<'EOF'> xfce4-terminal
+WAYLAND_DISPLAY=wayland-0
```
Comment (uberhacker) [inline packages/xfce4_terminal.rb:41]: Probably should add `#!/bin/bash` at the top.

Comment (supechicken) [reply packages/xfce4_terminal.rb:42]: Done

File: packages/xfce4_terminal.rb [hunk 4]
```diff
@@ -36,5 +37,13 @@ def self.build
...
+GDK_BACKEND=wayland
+DISPLAY=
+#{CREW_PREFIX}/bin/xfce4_terminal $@
```
Comment (uberhacker) [inline packages/xfce4_terminal.rb:45]: Probably should put quotes around `\"$@\"`.

Comment (supechicken) [reply packages/xfce4_terminal.rb:45]: No need, as `'EOF'` will escape all meaning of syntax automatically

Comment (uberhacker) [reply packages/xfce4_terminal.rb:45]: I'm talking about putting quotes around the arguments, not escaping the characters.

Comment (supechicken) [reply packages/xfce4_terminal.rb]: Done

**Pull request merged**